server/master/src/main/java/org/apache/accumulo/master/Master.java

```diff
@@ -1536,25 +1545,16 @@ private void getMasterLock(final String zMasterLoc) throws KeeperException, Inte
   }
 
   public static void main(String[] args) throws Exception {
+    final String app = "master";
+    ServerOpts opts = new ServerOpts();
+    opts.parseArgs(app, args);
+    ServerContext context = ServerContext.getInstance();
```

Comment:
Could the constructor for ServerContext be used here?  I think it would be nice to move away from using static singletons wherever possible in new code.

Reply:
It could but I think this needs to be addressed in later PR. I created #582